Description: merge duplicate NAME/VERSION sections in POD
 Usually caused by a Pod::Weaver accident, or some mixture
 of manual and automatic updates.
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2018-11-17
Forwarded: 
Bug: 

--- a/lib/Error.pm
+++ b/lib/Error.pm
@@ -682,7 +682,7 @@
 
 =head1 NAME
 
-Error
+Error - Error/exception handling in an OO-ish way
 
 =head1 VERSION
 
@@ -736,14 +736,6 @@
 Errors in the class C<Error> should not be thrown directly, but the
 user should throw errors from a sub-class of C<Error>.
 
-=head1 NAME
-
-Error - Error/exception handling in an OO-ish way
-
-=head1 VERSION
-
-version 0.17027
-
 =head1 WARNING
 
 Using the "Error" module is B<no longer recommended> due to the black-magical
--- a/lib/Error/Simple.pm
+++ b/lib/Error/Simple.pm
@@ -23,7 +23,7 @@
 
 =head1 NAME
 
-Error::Simple
+Error::Simple - the simple error sub-class of Error
 
 =head1 VERSION
 
@@ -42,14 +42,6 @@
 and the only thing it does is "use" Error.pm. Refer to the documentation
 of L<Error> for more information about Error::Simple.
 
-=head1 NAME
-
-Error::Simple - the simple error sub-class of Error
-
-=head1 VERSION
-
-version 0.17027
-
 =head1 METHODS
 
 =head2 Error::Simple->new($text [, $value])
